wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -213,8 +210,14 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
             nameDocuments.map { getIdentityForName(it) }
         }
 
-        val profileDocuments = Profiles(platform).getList(userIds)
-        val profileById = profileDocuments.associateBy({ it.ownerId }, { it })
+        var profileById: Map<Identifier, Document> = mapOf()
+
+        try {
+            val profileDocuments = Profiles(platform).getList(userIds)
+            profileById = profileDocuments.associateBy({ it.ownerId }, { it })
+        } catch (e: StatusRuntimeException) {
+            // swallow; we don't want to stop this method if no usernames have profiles
+        }
```

Comment:
if none of the search results have profiles, then an exception would be thrown.

This might actually be from a platform bug if a where "in" query has no results, then an exception is thrown.